src/richie/apps/courses/templates/courses/cms/_course_glimpses.scss

```diff
@@ -0,0 +1,194 @@
+///
```

Comment:
I don't think this is the correct location for this file.

Files relevant to a frontend component only => next to their component in `richie-front`
Files relevant to a backend template => next to their template

Shared files used by both frontend + backend => in the common `scss` folder next to `main.scss`